torch_xla/experimental/torch_triton.py

```diff
@@ -0,0 +1,205 @@
+"""Module for calling Triton kernels from Pytorch/XLA."""
+
+from __future__ import annotations
+
+import os
+from typing import Any, Callable, Dict, Tuple, Union
+import zlib
+import torch
+
+import numpy as np
+import triton
+import triton.language as tl
+import torch_xla.csrc.triton._triton as lib_triton
```

Comment:
So all the triton package you built is to access in python? If that's the case, then when can't we just use jax_lib?

Reply:
You're right! I justed verified that just using jaxlib should also work. I was earlier worried being able to register XLA custom call from JAX but that works too as it just uses a static registry object for custom calls. This should make this PR much simpler. I'll update it and request a review again.

Reply:
Perfect. Looking forward to it!